src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4057,12 +4089,17 @@ namespace SimAirServingZones {
 
 				for ( ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
 					CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
+					int TermUnitSizingIndex = AirToZoneNodeInfo( AirLoopNum ).TermUnitCoolSizingIndex( ZonesCooledNum );
```

Comment:
OK by me, for now. We can change the variable name later. I'm just trying to find issues before this drops in.